frontend/src/views/admin/Users/ExternalUsersView.vue

```diff
@@ -0,0 +1,126 @@
+<template>
+  <v-card class="table">
```

Comment:
not sure this is the correct UI, may be diaglog in the list of course executions